eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -24,11 +24,11 @@ jobs:
           OSName: "Windows"
           OSVmImage: "windows-2019"
           TestTargetFramework: netcoreapp2.1
-        Windows_NetCoreApp_ProjectReferences:
-          OSName: "Windows"
-          OSVmImage: "windows-2019"
-          TestTargetFramework: netcoreapp2.1
-          ConvertToProjectReferenceOption: /p:UseProjectReferenceToAzureClients=true
+        # Windows_NetCoreApp_ProjectReferences:
```

Comment:
We should definitely not comment this out. As we chatted in IM we should update the test projects.